oauthlib/oauth2/rfc6749/request_validator.py

```diff
@@ -270,6 +277,20 @@ def save_bearer_token(self, token, request, *args, **kwargs):
         """
         raise NotImplementedError('Subclasses must implement this method.')
 
+    def get_id_token(self, token, token_handler, request):
```

Comment:
In general it appears that the expectation has been that clients will be pulling any relevant parameters off of the `request`.  I'll make a note in the docs about the nonce in implicit and hybrid workflows.